astroid/builder.py

```diff
@@ -280,7 +280,9 @@ def delayed_assattr(self, node):
 
 
 def build_namespace_package_module(name: str, path: List[str]) -> nodes.Module:
-    return nodes.Module(name, doc="", path=path, package=True)
+    node = nodes.Module(name, path=path, package=True)
+    node.postinit(body=[], doc_node=nodes.Const(value=""))
```

Comment:
Or do we want to make `doc_node` `None` here?

Reply:
I think it would be safe to leave it as `None`. Code already needs to handle it for Modules created by the TreeRebuilder, so this wouldn't introduce anything new.

Since we no longer need the `postinit` call then, I suggest to return the Module node directly instead.